ckanext/pages/plugin/__init__.py

```diff
@@ -197,6 +198,9 @@ def get_auth_functions(self):
             'ckanext_group_pages_list': auth.group_pages_list,
         }
 
+    def configure(self, config):
+        db.init_db()
```

Comment:
If we are creating the tables explicitly with the command line we don't need to do that here right? Otherwise we have the same issue with multithreads that we were trying to avoid

Reply:
The ORM needs to be initialized anyway, similar structures are in [harvest](https://github.com/ckan/ckanext-harvest/blob/e2e8acbb6d87d44663e92c5ff5cbd67401588c2d/ckanext/harvest/plugin/__init__.py#L225-L226) and [spatial](https://github.com/ckan/ckanext-spatial/blob/5a987828cb8e76f4904c6f23542a593255171a94/ckanext/spatial/plugin/__init__.py#L86-L87).

Reply:
And just to confirm init_db only defines tables for ORM if the tables exist https://github.com/ckan/ckanext-pages/blob/6c4eb4711b746b86c2a83c57643c263219f554ce/ckanext/pages/db.py#L22-L27